package.json

```diff
@@ -92,6 +92,7 @@
     "Nicolas Gonzalez",
     "Pankaj Patil <pankaj.patil2099@hotmail.com> (https://twitter.com/pankajpatil16)",
     "Peter",
+    "Prince Wilson (https://twitter.com/maxcell)",
```

Comment:
I... didn't add this, I assume there's some automation happening somewhere 😄

Reply:
Yes just ignore it :), we need to fix the flow to actually do these changes in the related PRs instead of in the next PR.